### DISCUSSION TITLE
Supporting suggestion SpaceLocation parent

### DISCUSSION THREAD

**Discussion by YPetremann** [notnotmelon/factorissimo-2-notnotmelon#112] [General]
I've made a suggestion to add parent to SpaceLocations so multiple stars could be added with their set of planets orbiting around and moons having orbit around their parent planet
https://forums.factorio.com/viewtopic.php?f=28&t=121788

It would be great to give support to this suggestion since your mod kind of add "satelites" to planets, it would ease the creating of them.

**Comment by notnotmelon** [comment]
Thanks!
